tests/ui/impl-trait/in-trait/missing-type-parameter.rs

```diff
@@ -0,0 +1,14 @@
+// [next] compile-flags: -Zlower-impl-trait-in-trait-to-assoc-ty
+// revisions: current next
+
+#![feature(return_position_impl_trait_in_trait)]
+
+struct Wrapper<G: Send>(T);
+//~^ ERROR: cannot find type `T` in this scope [E0412]
```

Comment:
This resolver error is not necessary. Did you try what I told you like an hour ago and try to reduce the test so that it doesn't include spurious errors?

Reply:
Ok, this is right. I've modified it but I don't think it can be reduced further.